### PR TITLE
Reduce default fees

### PR DESCRIPTION
**Pull request by rustyrussell**

### CODE REVIEW

**Review by vincenzopalazzo — APPROVED** [review]
utACK https://github.com/ElementsProject/lightning/pull/4507/commits/19e89fcaee0354c31ac7c49934944dcb909b0feb

File: plugins/bcli.c [hunk 1]
```diff
@@ -694,32 +594,97 @@ static struct command_result *getchaininfo(struct command *cmd,
 	return command_still_pending(cmd);
 }
 
+/* Mutual recursion. */
+static struct command_result *estimatefees_done(struct bitcoin_cli *bcli);
+
+/*
+ * Calls `estimatesmartfee` with targets 2/CONSERVATIVE (very urgent),
+ * 3/CONSERVATIVE (urgent), 4/ECONOMICAL (normal), and 100/ECONOMICAL (slow)
```
Comment (niftynei) [inline plugins/bcli.c:602]: this comment should be updated in your next commit

**Review by niftynei — APPROVED** [review]
ACK 19e89fcaee0354c31ac7c49934944dcb909b0feb

File: plugins/bcli.c (hunk 1 above)
Comment (rustyrussell) [reply plugins/bcli.c]: I deleted it.  The code is now clear, and the comment just risks getting outdated again.

**Comment by rustyrussell** [comment]
OK, this is tripping dualopend confusion with connectd getting desynchronized on disconnect messages.  Adding debugging now...

**Pull request merged**